tests/drivers/gpio/gpio_api_1pin/prj.conf

```diff
@@ -1,2 +1,3 @@
 CONFIG_ZTEST=y
 CONFIG_GPIO=y
+CONFIG_GPIO_GET_CONFIG=y
```

Comment:
Why enable it here? This test does not make use of the API?

Reply:
Accidentally added to wrong file, my bad.